Nats_Subscriber/NatsSubscriber.cs

```diff
@@ -48,7 +51,10 @@ public async void GetStream()
         {
             try
             {
-                NatsConnection nats = new NatsConnection();
+                NatsConnection nats = new NatsConnection(new NatsOpts
+                {
+                    SubPendingChannelFullMode = BoundedChannelFullMode.Wait,
```

Comment:
this will make sure we won't drop in coming messages. by default on a busy subscription if we can't process it fast enough we drop the message which is probably not making much sense (as I'm typing this) but it might have an effect overall.